### PR TITLE
Bubble Sort 

### PR DESCRIPTION
**Pull request by Eshy10**
## Bubble Sort Using Ruby
- Added two methods to bubble sort an array
- The second bubble_sort_by method uses a yield
- The block is used to return value to sort an array

### CODE REVIEW

File: bubble_sort.rb
Comment (mosesogwo) [inline bubble_sort.rb]: Return the arr, don't print it.

File: bubble_sort.rb
Comment (mosesogwo) [inline bubble_sort.rb]: Return the array. Don't print it

File: bubble_sort.rb
Comment (mosesogwo) [inline bubble_sort.rb]: Remove your test example.

File: bubble_sort.rb
Comment (mosesogwo) [inline bubble_sort.rb]: Remove your test example

**Review by mosesogwo — CHANGES_REQUESTED** [review]
**Hi @Eshy10, @vzdrizhni. Nice job on this project:+1:. A few fixes are required for approval :fast_forward:**

1. Revert .stickler.yml to the original config. You are not to modify the content of the linter config files.

2. Modify your methods to return the sorted arrays instead of printing them.

3. You should remove your own test examples from the file.

_**Kindly make these changes and submit another code review request.**_

**Cheers:stars:**

### Status: Changes required!

File: bubble_sort.rb
Comment (Eshy10) [reply bubble_sort.rb]: The method uses a yield removing it cant break the yield. I am not sure I understand what you meant by removing the example here.

Comment (mosesogwo) [reply bubble_sort.rb]: You have tested your bubble_sort_by on the ['hello', 'hi', 'hey'] enum, right?

**Review by fatymahmed — APPROVED** [review]
## Hi @Eshy10 @vzdrizhni,
### Great job making the suggested changes🎉🎉
- Before merging, check my comment below.
### It's a 👍🏾 from me!
## ✔️ Approved!

Good luck with your next projects.
 _Don't forget to submit your project completion form_

File: bubble_sort.rb
Comment (fatymahmed) [inline bubble_sort.rb]: - Please remove this part, you can move it to another file for testing purpose.

Comment (Eshy10) [reply bubble_sort.rb]: okay done. thanks

**Pull request merged**
